src/util/mod.rs

```diff
@@ -39,6 +39,7 @@ pub(crate) mod erase_vm;
 pub(crate) mod finalizable_processor;
 /// Heap implementation, including page resource, mmapper, etc.
 pub mod heap;
+/// Checking if an address is an valid MMTk object.
```

Comment:
I am considering moving those doc comments into their respective modules in the form of `//!` at the beginning of the file so that we can have more detailed documentation for each module.  But we may do it in a separate PR.

Reply:
I added the comments in a format that is consistent with the other comments in the same file. If we want to move the docs to the modules, we should systematically do so for the code base. I don't plan to do that in the PR.